pyproject.toml

```diff
@@ -59,31 +59,21 @@ path = "dbt/adapters/postgres/__version__.py"
 dependencies = [
     "dbt-adapters @ git+https://github.com/dbt-labs/dbt-adapters.git",
     "dbt-common @ git+https://github.com/dbt-labs/dbt-common.git",
-]
-
-[tool.hatch.envs.lint]
```

Comment:
There's no point in separate lint and typecheck environments. We want the default to be able to run precommit, which requires all of this in default.